pkg/qemu

```diff
@@ -1,10 +1,10 @@
 [mirrors]
```

Comment:
please move the old qemu recipe to pkg/qemu281

all the patch filenames need to be prefixed with `qemu5-` otherwise the stuff in KEEP will become confusing